services/migrations/gitea_uploader.go

```diff
@@ -109,6 +109,7 @@ func (g *GiteaLocalUploader) CreateRepo(repo *base.Repository, opts base.Migrate
 		return err
 	}
 	r.DefaultBranch = repo.DefaultBranch
+	r.Description = opts.Description
```

Comment:
Or we could not change below, but only change this to `r.Description = repo.Description`?

Reply:
oooh good one ! You're totally right...